gym/utils/env_checker.py

```diff
@@ -165,47 +196,49 @@ def check_reset_options(env: gym.Env):
                 f"The error was: {e}"
```

Comment:
So yes this will work, in that it will be `True` in the function contains an argument `kwargs` however this code doesn't check if this is `**kwargs` or `kwargs`. I have updated it with `signature.parameters["kwargs"].kind is inspect.Parameter.VAR_KEYWORD` that should check if it is `**kwargs` correctly.

Reply:
Thanks, I guess that's good enough although, strictly speaking, it will not catch something like `**kwparam`, which would also be valid

Reply:
I could change it to be if any of the parameter are of kind VAR_KEYWORD?